src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -44,7 +43,6 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('tags', arg_type=tags_type)
         c.argument('registry_name', options_list=['--name', '-n'], help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr')
         c.argument('tenant_suffix', options_list=['--suffix'], help="The tenant suffix in registry login server. You may specify '--suffix tenant' if your registry login server is in the format 'registry-tenant.azurecr.io'. Applicable if you\'re accessing the registry from a different subscription or you have permission to access images but not the permission to manage the registry resource.")
-        c.argument('storage_account_name', help='Provide the name of an existing storage account if you\'re recreating a container registry over a previous registry created storage account. Only applicable to Classic SKU.', completer=get_resource_name_completion_list(STORAGE_RESOURCE_TYPE))
         c.argument('sku', help='The SKU of the container registry', arg_type=get_enum_type(SkuName))
```

Comment:
Remove classic from the --sku list

Reply:
`SkuName` is generated by sdk and contains `Classic` because users can still get Classic SKU. It will be removed soon from swagger. Do you want to process and remove it separately for now?

Reply:
Discussed offline, will keep it this way for now and let the next sdk remove it once deprecation is completely done.